docs/content/latest/explore/multi-region-deployments/follower-reads-ysql.md

```diff
@@ -9,25 +9,247 @@ menu:
     parent: explore-multi-region-deployments
-    weight: 285
+    weight: 280
 isTocNested: true
 showAsideToc: true
 ---
 
 <ul class="nav nav-tabs-alt nav-tabs-yb">
-  
   <li >
     <a href="../follower-reads-ysql/" class="nav-link active">
       <i class="icon-postgres" aria-hidden="true"></i>YSQL</a>
   </li>
-  
+
   <li >
     <a href="../read-replicas-ycql/" class="nav-link">
       <i class="icon-cassandra" aria-hidden="true"></i>YCQL</a>
-  </li>  
-
+  </li>
 </ul>
 
-YSQL follower reads support is currently in Beta. 
+## Introduction
+
+YugabyteDB requires reading from the **leader** to read the latest data. However, for applications that don't require the latest data, or are working with unchanging data, the cost of contacting a potentially remote leader to fetch the data may be wasteful. Your application may benefit from better latency by reading from a **replica** that is closer to the client.
+
+Replicas may not be completely up to date with all updates, so this design may respond with stale data. You can specify how much staleness the application can tolerate. When enabled, read-only operations may be handled by the closest replica, instead of having to go to the leader.
+
+## Surface area and usage
+
+### Surface area
+
+Two session variables control the behavior of follower reads:
+
+- `yb_read_from_followers` controls whether reading from followers is enabled. Default is false.
+- `yb_follower_read_staleness_ms` sets the maximum allowable staleness. Default is 30 seconds.
+
+### Expected behavior
+
+The table describes what the expected behavior is when a read happens from a follower.
+
+| Conditions | Expected behavior |
+| :--------- | :---------------- |
+| yb_read_from_followers is true AND transaction is marked read-only | Read happens from the closest follower |
+| yb_read_from_followers is false OR transaction/statement is not read-only | Read happens from the leader |
+
+### Read from follower conditions
+
+- If the follower’s safe-time is at least `<current_time> - <staleness>`, then the follower may serve the read without any delay.
+
+- If the follower is not yet caught up to `<current_time> - <staleness>`, then the read will be redirected to a different replica transparently from the end-user. The end user may see a slight increase in latency depending on the location of the replica which satisfies the read.
+
+### Read-only transaction conditions
+
+Regarding marking the transaction as read only, a user can do one of the following:
+
+- `SET TRANSACTION READ ONLY` applies only to the current transaction block.
+- `SET SESSION CHARACTERISTICS AS TRANSACTION READ ONLY` applies the read-only setting to all statements and transaction blocks that follow.
+- `SET default_transaction_read_only = TRUE` applies the read-only setting to all statements and transaction blocks that follow.
+- `/*+ Set(transaction_read_only true) */ SELECT` applies only to the current transaction/select statement.
```

Comment:
Use [pg_hint_plan](https://docs.yugabyte.com/latest/explore/query-1-performance/pg-hint-plan/) mechanism	to embed the hint along with the SELECT statement. For example,` /*+ Set(transaction_read_only true) */ SELECT ...`  applies only	to the current SELECT  statement.